presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -94,12 +94,14 @@ public void testAccessControlCreateTableAsSelect()
     {
         String createTableAsSelect = "bob_create_table_as_select";
 
+        bobExecutor.executeQuery("DROP TABLE IF EXISTS " + createTableAsSelect);
```

Comment:
Can you please also do the same for:
 - testAccessControlCreateTableAsSelect
 - testAccessControlCreateView

Reply:
This _is_ in `testAccessControlCreateTableAsSelect`.

`testAccessControlCreateView`, `testAccessControlDropView` - updated